src/pages/database-main/DatabaseContent.jsx

```diff
@@ -4,7 +4,7 @@ import { useDatabaseContext } from "./DatabaseProvider";
 import RootLayout from "../../components/layout/RootLayout";
 import ContentSiderLayout from "../../components/layout/ContentSiderLayout";
 import { Link } from "react-router-dom";
-import { Divider } from "antd";
+import { Input, Divider } from "antd";
```

Comment:
inaccurate component to use.

for search bar implementation using antd, we are most likely going to implement it using `Select`

reference:
https://ant.design/components/select#select-demo-search-box